src/ticket.js

```diff
@@ -37,52 +35,55 @@ class Ticket {
     return update.then(() => this.save());
   }
 
-  setAgent(userId) {
+  setAgent (userId) {
     debug('setting agent');
-    return users.find(userId).then((result) => {
```

Comment:
Hi! I think you may have removed this by accident! This is still needed to make the "Agent" / assignee field to work properly.

Reply:
@girliemac I removed that on purpose. I'm using a `users_select` and `user_id` in a formatted message instead of plain text user's name.